### DISCUSSION TITLE
CT24

### DISCUSSION THREAD

**Discussion by darrencycheung** [iptv-org/community#833] [Q&A]
### Channel ID

CT24.cz

### Notes

Valid m3u8 plz

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/cz.m3u#L8-L9
